site/media/css/main.css

```diff
@@ -250,10 +251,84 @@ ul#cats a:hover {
 #cats .location { background-image: url('/media/images/categories/location.png'); }
 #cats .transparency { background-image: url('/media/images/categories/transparency.png'); }
 
+#cats .inform { background-image: url('/media/images/categories/inform.png'); }
+#cats .notify { background-image: url('/media/images/categories/defaultb.png'),
+  linear-gradient(0deg, #EDF5FF, #EDF5FF); }
```

Comment:
All are intended to be light enough to read text on. They are the colors used in a diagram we have been working on showing all the privacy patterns. The colors are based on the strategy the tactic falls under:
- Control: Yellow / cream
- Hide: orange / terra cotta
- Enforce: purple
- Inform: dark blue
- Demonstrate: light blue
- Abstract: blue green / turquoise
- Separate: green
- Minimize: green yellow / lime